### PR TITLE
Test dextrose fix

### PR DESCRIPTION
**Pull request by L0wry**

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/3f8cb0c9cd580d5554cd6cb7671cf70b6f1e401d/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/3044f0eb67d41f55a48ef98df39ef99eb39120c5/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/41efee9889110f145060ca5af9a4e0fc1cce6e5a/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/1e33b05d85518aaca6fdeb464553563e22d6e503/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/15d82e663f5e659a19ddf50ab96a3e12c91d85b8/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/47d84a36633ac59dd427d309b3732822030cc37c/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/824c10d6b883f1b0f77e4a077d25c26616b2cf76/dextrose-test/index.html

File: packages/article-byline/package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "@storybook/react-native": "3.3.11",
     "@times-components/eslint-config-thetimes": "0.1.3",
     "@times-components/jest-configurator": "0.2.0",
-    "dextrose": "1.4.5",
+    "dextrose": "^1.4.6",
```
Comment (asifhafeez) [inline packages/article-byline/package.json]: Shouldn't we be specifying a specific version rather than using '^'?

**Pull request merged**